packages/relay/src/lib/eth.ts

```diff
@@ -542,6 +543,19 @@ export class EthImpl implements Eth {
     return gas;
   }
 
+  /**
+   * Perform value format precheck before making contract call towards the mirror node
+   * @param transaction 
+   */
+  contractCallPrecheck(transaction: any) {
```

Comment:
nit: not really a precheck more of a format
```suggestion
  contractCallFormat(transaction: any) {
```
or 
```suggestion
  formatCcontractCall(transaction: any) {
```

Reply:
Changed.